src/briefcase/platforms/android/gradle.py

```diff
@@ -180,6 +200,7 @@ def output_format_template_context(self, app: AppConfig):
                 for path in (app.test_sources or [])
                 if (name := Path(path).name)
             ),
+            "dependencies": {"implementation": dependencies},
```

Comment:
Since we won't be using any of the other dependency types in the foreseeable future, adding an extra `implementation` layer seems unnecessary, and loses the simplicity of having the cookiecutter context match directly with the pyproject.toml structure.

Reply:
Unfortunately, it is needed. Cookiecutter doesn't let you pass a list as a context value - if you do, it's interpreted as a choice, and it gives you the "default" (the first item). [Passing a dictionary with a single key](https://github.com/cookiecutter/cookiecutter/issues/479#issuecomment-842224554) is how you work around this; in this case, it just happens that there is a convenient name available for that key, even if we're not going to use any of the alternative key values.